python/gtsam/examples/SFMExample_bal.py

```diff
@@ -7,112 +7,122 @@
   See LICENSE for the license information
 
   Solve a structure-from-motion problem from a "Bundle Adjustment in the Large" file
-  Author: Frank Dellaert (Python: Akshay Krishnan, John Lambert)
+  Author: Frank Dellaert (Python: Akshay Krishnan, John Lambert, Varun Agrawal)
 """
 
 import argparse
 import logging
 import sys
 
-import matplotlib.pyplot as plt
-import numpy as np
-
 import gtsam
-from gtsam import (
-    GeneralSFMFactorCal3Bundler,
-    PinholeCameraCal3Bundler,
-    PriorFactorPinholeCameraCal3Bundler,
-    readBal,
-    symbol_shorthand
-)
+from gtsam import (GeneralSFMFactorCal3Bundler,
+                   PriorFactorPinholeCameraCal3Bundler, PriorFactorPoint3)
+from gtsam.symbol_shorthand import C, P
+from gtsam.utils import plot
```

Comment:
I would import this as something like
```python
import gtsam.utils.plot gtsam_plot_utils
```

or 
```python
import gtsam.utils.plot gtsam_plot
```

Otherwise I get confused about whether this is matplotlib

Reply:
Which is why I had the functions imported directly. :slightly_smiling_face: 
If we are okay with that, I can revert to the function import, since this is the case everywhere in the examples.

Reply:
I think if it's wrong in the other examples, we can slowly make the correct code go viral, starting from here : - )

Importing functions should not be allowed. But if you prefer calling it `plot`, feel free to leave this change as you have it here

Reply:
What's the reason for that? Google's style guide mentions `Use import statements for packages and modules only, not for individual classes or functions`, so we're clearly not abiding by this. :slightly_smiling_face: 

Plus, I am of the opinion that these are _guidelines_ and not the rules, so we can not conform to them if it means the code is more readable.

Reply:
Also, ironically, calling it `gtsam_plot_utils` is no better than simply doing `gtsam.utils.plot.whatever_func`.

Reply:
I think many modern Python libraries like Pytorch abide by this. Otherwise it's hard to immediately tell where to go to look for the implementation of a function (do I ctrl-F in this file, or look in another file)

 Interesting that Google style forbids the class import, though.

Reply:
Pytorch does something way wackier. Plus they have a bunch of people making 200K a year for this stuff, no 22K a year like yours truly. :slightly_smiling_face: 

If you paid me that much money, I'd be more than happy to accommodate this request. :joy: